server/tsuserver.py

```diff
@@ -58,7 +58,7 @@ def __init__(self):
         self.district_client = None
         self.ms_client = None
         self.rp_mode = False
-        logger.setup_logger(debug=self.config['debug'])
+        logger.setup_logger(debug=self.config['debug'], log_size=self.config['log_size'], log_backups=self.config['log_backups'])
```

Comment:
This is not enough to read from the config. You need to specify default values in `load_config` I believe - otherwise, it will result in a `KeyError` for players who have not added the two new config entries.

Reply:
Done.